app/src/main/java/io/github/droidkaigi/confsched/util/DataBindingAttributeUtil.java

```diff
@@ -84,4 +89,26 @@ public static void setSessionFab(FloatingActionButton fab, @NonNull Session sess
         fab.setSelected(session.checked);
     }
 
+    @BindingAdapter("forceRtlDirection")
+    public static void setForceRtlDirection(FlowLayout flowLayout, boolean isCenterVertical) {
```

Comment:
FlowLayout is used as tags container.
This view does not support RTL. So I created this custom attributes.